### PR TITLE
update to latest ledger protos

### PR DESCRIPTION
**Pull request by jin-tron**
SignedCreateAccount was not in ledger proto, add it now.

### CODE REVIEW

**Comment by wxue** [comment]
@jin-tron Do you want to merge in my PR first tomorrow? I figured out the way to fix timestamp -> time issue, will wrap up with gogo plugin integrated. So you can regenerate your PB by using that.

**Comment by jin-tron** [comment]
that's good, I will wait for you PR.

File: ledger/ledger.go [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+IEdyb3VwLCBJbmMuMTEwLwYDVQQLEyhHbyBEYWRkeSBDbGFzcyAyIENlcnRpZmlj
+YXRpb24gQXV0aG9yaXR5ggEAMAwGA1UdEwQFMAMBAf8wDQYJKoZIhvcNAQEFBQAD
+ggEBADJL87LKPpH8EsahB4yOd6AzBhRckB4Y9wimPQoZ+YeAEW5p5JYXMP80kWNy
+OO7MHAGjHZQopDH2esRU1/blMVgDoszOYtuURXO1v0XJJLXVggKtI3lpjbi2Tc7P
+TMozI+gciKqdi0FuFskg5YmezTvacPd+mSYgFFQlq25zheabIZ0KbIIOqPjCDPoQ
+HmyW74cNxA9hi63ugyuV+I6ShHI56yDqg+2DzZduCLzrTia2cyvk0/ZM/iZx4mER
+dEr/VxqHD3VILs9RaRegAhJhldXRQLIQTO7ErBBDpqWeCtWVYpoNz4iCxTIM5Cuf
+ReYNnyicsbkqWletNw+vHX/bvZ8=
+-----END CERTIFICATE-----`
+)
```
Comment (wxue) [inline ledger/ledger.go]: is the certificate okay to be shown here?

Comment (wxue) [reply ledger/ledger.go]: I'd suggest put in another file and import in.

Comment (ShellyWEI) [reply ledger/ledger.go]: We actually don't have another way to talk to ledger for now. It's more like a public key. But we can config it in some way on the instance running program for sure.

Comment (taiyangc) [reply ledger/ledger.go]: FWIW they changed the dev domain as well - better to have this read from ENV variables. Don't really need to commit it.

File: ledger/ledger.go [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+func LedgerConnection() (*grpc.ClientConn, error) {
```
Comment (taiyangc) [inline ledger/ledger.go]: ledger addr and cert passed into this func

Comment (jin-tron) [reply ledger/ledger.go]: updated, read from env

File: ledger/ledger.go (hunk 1 above)
Comment (jin-tron) [reply ledger/ledger.go]: read from env

File: ledger/ledger.go [hunk 3]
```diff
@@ -157,7 +131,7 @@ func CreateChannel(ctx context.Context, ledgerClient ledgerPb.ChannelsClient, ch
 }
 
 func CloseChannel(ctx context.Context, signedChannelState *ledgerPb.SignedChannelState) error {
-	clientConn, err := LedgerConnection()
+	clientConn, err := LedgerConnection(os.Getenv("LEDGER_ADDR"), os.Getenv("LEDGER_CERT"))
```
Comment (taiyangc) [inline ledger/ledger.go]: `CloseChannel()` should not decide how to read env variables. It's up to the caller of this module to decide.

I think this function should follow how `CreateChannel` works but taking in the `ledgerClient`

A follow up might be to re-factor this to have all methods on the ledger client object cc @ShellyWEI

Comment (jin-tron) [reply ledger/ledger.go]: fixed, now add ledgerClient in input parameter, will leave the called to get the ledgerClient and then call this CloseChannel function.

**Comment by taiyangc** [comment]
Needs one more approval @ShellyWEI @wxue

**Pull request merged**